back/server.js

```diff
@@ -37,4 +41,30 @@ app.get('/offer/:id', (req, res) => {
   });
 });  
 
+// POST OFFERS ////////////////////////////////////////////////
+app.post('/offer/add', (req, res) => {
```

Comment:
REST conventions: URLs use plural for resource names:

    /offers/add

Check all endpoints